Services/Init/classes/class.ilPasswordAssistanceGUI.php

```diff
@@ -636,47 +636,14 @@ public function submitUsernameAssistanceForm()
 			return;
 		}
 
-		// Retrieve form data
-		$email = $form->getInput('email');
-
-		// Retrieve a user object with matching user name and email address.
+		$email  = $form->getInput('email');
 		$logins = ilObjUser::_getUserIdsByEmail($email);
 
-		// No matching user object found?
-		// Show the password assistance form again, and display an error message.
-		if(!is_array($logins) || count($logins) < 1)
-		{
-			ilUtil::sendFailure(str_replace("\\n", '', $this->lng->txt('pwassist_invalid_email')));
```

Comment:
Could we remove `pwassist_invalid_email`?

Reply:
I removed this.